wpeverest/everest-forms
# Fix - Form submissions issues

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixed the form submission issue with minimum values and no required. See [ticket](https://app.crisp.chat/website/edf6307a-f7f2-4c60-834f-5423a00d673b/inbox/session_1ac3ac11-fb7e-4765-a0f1-78fe4eee15e1/)

### How to test the changes in this Pull Request:

1. Set Min Max values
2. Set not required
3. Submit Entries
4. Verify its working

### Types of changes:




### Other information:




### Changelog entry

> Fix - Form submissions issues
